Examples/Pass/arc.cvf

```diff
@@ -44,16 +34,12 @@ method drop() {
       {| noCnt() |}
         <free = (true)>;
       {| emp |}
-    } else {
-      {| emp |}
-        dummy = dummy; // skip;
-      {| emp |}
     }
   {| emp |}
 }
 
 view error();
```

Comment:
TBF doing it here will make the PLDI muddle a tiny bit more understandable and easier to pull apart, at the expense of having to twiddle the change back into working with the PLDI version of the AST.